pylint_odoo/checkers/modules_odoo.py

```diff
@@ -811,3 +817,21 @@ def _check_file_not_used(self):
         if self.msg_args:
             return False
         return True
+
+    def _check_xml_attribute_translatable(self):
+        """The xml attribute is missing the translation="off" tag
+            Example  <attribute name="groups">sale.group</attribute>
+        """
+        self.msg_args = []
+        for xml_file in self.filter_files_ext('xml', relpath=True):
+            doc = self.parse_xml(os.path.join(self.module_path, xml_file))
+            records = doc.xpath(
+                "/openerp//record//attribute[not(@translation)]") + \
```

Comment:
But you could append a parameter to the get_xml_records method to make it more direct!
Like this:
```python
def get_xml_records(self, xml_file, model=None, more=''):                   
    .
    .
    .
    return doc.xpath("/openerp//record" + model_filter + more) + \          
    doc.xpath("/odoo//record" + model_filter + more) \                                                                                                     
    if not isinstance(doc, basestring) else [] 
    .
    .
    . 
```
The parameter is appended more!
`more = '//attribute[not(@translation)]'`